pkg/users/account-create-dialog.js

```diff
@@ -42,6 +42,16 @@ function AccountCreateBody({ state, errors, change }) {
         locked, change_passw_force
     } = state;
 
+    // We want to let user know that password and confirmation password do not match without them having to constantly click on "Create" to validate the form.
+    // But we also don't want to show an error message while they are typing the confirmation password, only when they are finished typing it.
+    // To solve the issue of telling that user is finished writing the confirmation password, let's do the following:
+    // Since both passwords should be the same length, we should show error message while user is typing,
+    // but only if the confirm password is same length (or higher) than the original password (i.e. they are finished typing)
+    let dynamic_password_confirm_error;
+    if (state.password_confirm.length >= state.password.length)
+        dynamic_password_confirm_error = validate_password_confirm(state.password_confirm, state.password);
+    console.log(dynamic_password_confirm_error);
```

Comment:
forgotten debugging output

Reply:
Ah thanks, I see it made the test output unreadable.